vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2219,6 +2219,7 @@ def _dummy_run(
         uniform_decode: bool = False,
         skip_eplb: bool = False,
         is_profile: bool = False,
+        remove_lora: bool = True,
```

Comment:
Please add a docstring for the arg, especially if we might remove it in the future